gradle.properties

```diff
@@ -4,3 +4,9 @@ nexusUrlSnapshots=http://bale:8081/service/local/repositories/snapshots/content
 nexusUsername=admin
 nexusPassword=
 target=dev
+
+# cmo-metadb nats publishing properties
```

Comment:
I noticed that there's actually an `app.properties` - I'll move these props over to there